visual-dotnet/SauceLabs.Visual/Utils/IgnoreRegions.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections.Generic;
+using System.Linq;
+using OpenQA.Selenium;
+using SauceLabs.Visual.GraphQL;
+using SauceLabs.Visual.Models;
+
+namespace SauceLabs.Visual.Utils
+{
+    internal class IgnoreRegions
+    {
+
+        internal static void SplitIgnoredRegions(SelectiveRegion[]? regions, IgnoreRegion[]? ignoreRegions, IWebElement[]? ignoreElements, out RegionIn[] regionIns, out ElementIn[] elementIns)
```

Comment:
Interesting approach but I would recommend returning an object with regionIns and elementIns that can be destructed later